src/vfile_kind.mli

```diff
@@ -12,20 +12,16 @@ module type S = sig
   val id : t Id.t
 
   val load : Path.t -> t
-  val to_string : Path.t -> t -> string
+  val to_string : t -> string
 end
 
 type 'a t = (module S with type t = 'a)
 
 val eq : 'a t -> 'b t -> ('a, 'b) eq option
 
 module Make
-    (T : sig type t end)
-    (F : functor (C : Sexp.Combinators) -> sig val t : T.t C.t end)
-  : S with type t = T.t
-
-module Make_full
-    (T : sig type t end)
-    (To_sexp : sig val t : Path.t -> T.t -> Sexp.t end)
-    (Of_sexp : sig val t : Path.t -> Sexp.Ast.t -> T.t end)
+    (T : sig
+       type t
+       val t : t Sexp.To_sexp.t
```

Comment:
There should be a `name` in here as well, otherwise we won't be able to distinguish the various files for `dune dump`.